### DISCUSSION TITLE
Icon Redesign

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#63] [Team Posts]
The icons I am using for risk-first are from [The Noun Project](https://thenounproject.com), which is an excellent resource for icons, and well worth it.

However, randomly choosing icons from that site doesn't lead to a very coordinated selection of icons, so I have asked my brother (a professional graphic designer)to try and make them look a bit more consistent. 

**Original Risk Icons:**

![Original Periodic Table of Risks](https://riskfirst.org/images/generated/staging-and-classifying/periodic-table-large-400dpi.png)

**Some redesigned icons:**

![Sample Redesigned Images](https://riskfirst.org/images/generated/staging-and-classifying/redesign-400dpi.png)

**Comment by robmoffat** [comment]
Well, new icons are done now.  The image above got updated to show them, as are the rest of the images on riskfirst.org  

Hope they meet with everyone's approval.


cheers

**Comment by gandrade** [comment]
Looks good to me! Good job! :+1:

**Comment by robmoffat** [comment]
Cheers Gustavo!

**Comment by hovancik** [comment]
I don't see new icons, though. 404

**Comment by robmoffat** [comment]
All the images in my post are the redesigned versions, but I also fixed the URL on the second image.  How's that?

**Comment by hovancik** [comment]
👍
